app/scripts/modules/core/src/managed/ManagedResourceDetailsIndicator.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { get } from 'lodash';
+
+import { IEntityTags } from 'core/domain';
+import { HoverablePopover } from 'core/presentation';
+
+import './ManagedResourceDetailsIndicator.less';
+
+export const MANAGED_BY_SPINNAKER_TAG_NAME = 'spinnaker_ui_notice:managed_by_spinnaker';
+
+export interface IManagedResourceDetailsIndicatorProps {
+  entityTags: IEntityTags[];
+}
+
+export const ManagedResourceDetailsIndicator = ({ entityTags }: IManagedResourceDetailsIndicatorProps) => {
+  const isManaged =
+    get(entityTags, 'length') &&
+    entityTags.some(({ tags }) => tags.some(({ name }) => name === MANAGED_BY_SPINNAKER_TAG_NAME));
+
+  if (!isManaged) {
+    return null;
+  }
+
+  const helpText = (
+    <>
+      <p>
+        <b>Spinnaker is continuously managing this resource.</b>
+      </p>
+      <p>Changes made in the UI will be stomped in favor of the existing declarative configuration.</p>
```

Comment:
Is there anything we can link to here? Unless you already know about keel and declarative delivery, it's not really clear what any of this means.

Reply:
Indeed, a 'what on earth is this referring to?' doc on spinnaker.io for this tooltip is on the list to write in the near-term. For now anything keel-related is restricted to a few test apps as we work out documentation and some other rough edges.